sihl_users/lib/handler.ml

```diff
@@ -172,119 +172,67 @@ end
 
 module AdminUi = struct
   module Dashboard = struct
-    open Tyxml.Html
     open Sihl_core
 
-    let mycontent =
-      div
-        ~a:[ a_class [ "content" ] ]
-        [ h1 [ txt "A fabulous title" ]; txt "This is a fabulous content." ]
+    let handler =
+      Sihl_core.Http.get "/admin/dashboard/" @@ fun req ->
+      let user = Middleware.Authn.authenticate req in
+      Admin_ui.dashboard user |> Admin_ui.render |> Http.Response.html
+      |> Lwt.return
```

Comment:
After looking at this and similar formatting, shouldn't this be either:
```
      Admin_ui.dashboard user |> Admin_ui.render |> Http.Response.html
        |> Lwt.return
```
or
```
      Admin_ui.dashboard user
      |> Admin_ui.render
      |> Http.Response.html
      |> Lwt.return
```

Reply:
Ocamlformat takes care of all code formatting.